subprojects/docs/src/docs/userguide/about_manual.adoc

```diff
@@ -31,5 +31,5 @@ The Gradle User Manual is broken down into the following sections:
 
 . Gradle API http://gradle.org/docs/current/javadoc/[Javadocs]
 . Gradle's https://docs.gradle.org/current/dsl/index.html[Groovy DSL]
-. Gradle's https://gradle.github.io/kotlin-dsl-docs/api/[Kotlin DSL]
+. Gradle's https://docs.gradle.org/current/kotlin-dsl/index.html[Kotlin DSL]
```

Comment:
These shouldn't use absolute links to our docs because these will go to the wrong version when deployed.

I think these should all be relative: 
. Gradle API link:{javadocPath}/index.html[Javadocs]
. Gradle's link:{groovyDslPath}/index.html[Groovy DSL]
. Gradle's link:{kotlinDslPath}/index.html[Kotlin DSL]

I created a PR for this: 
https://github.com/gradle/gradle/pull/25170

Reply:
Thanks @big-guy!